.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,135 @@
+# Contributing to Moira
+
+First off, thanks for taking the time to contribute!
+
+The following is a set of guidelines for contributing to Moira and its packages, which are hosted in the 
+[Moira](https://github.com/moira-alert) Organization on GitHub. These are mostly guidelines, not rules. Use your best 
+judgment, and feel free to propose changes to this document in a pull request.
+
+**Table of Content**
+
+* [How Can I Contribute?](#how-can-i-contribute)
+    * [Submitting an Issue](#submitting-an-issue)
+    * [Contributing a Code](#contributing-a-code)
+* [Style Guides](#style-guides)
+    * [Git Style Guide](#git-style-guide)
+    * [Go Code Style Guide](#go-code-style-guide)
+    * [Testing Strategy](#testing-strategy)
+* [Development](#development)
+
+## How Can I Contribute?
+
+### Submitting an Issue
+
+If you find bugs or you want propose an idea that you think will be suitable for moira you can create an issue. The main 
+issue tracker is https://github.com/moira-alert/moira/issues. Issues that belongs for moira backend and web are placed here. 
+Issues that are related to python client and ansible role should be placed in issue tracker for related repo.
```

Comment:
What about docs?

Reply:
Still nothing about docs :-)

Reply:
I added a whole new section about docs. I do not understand what you mean here :(

Reply:
1. Issues that belong to backend and web are placed here.
2. Issues that are related to python client and ansible role should be placed there.

What about issues that belong to docs?

Reply:
Also, `belongs for` → `belong to`.

Reply:
I added a couple of words about docs and also placed couple of links for issue trackers.